aim/config.py

```diff
@@ -32,18 +32,6 @@
 
 LOG = logging.getLogger(__name__)
 
-global_opts = [
```

Comment:
[0] should now address both cases

[0] https://review.openstack.org/#/c/412022/